exercises/Tests-R.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: exercise
+title: Tests
+language: R
+---
+
+This is a follow up to the [Debugging problem]({{ site.baseurl }}/exercises/Debugging-R).
+
+Write tests for your 'Debugging' code for the following cases and save it in a 
+file called `Tests-R.R`. Make any corrections/improvements that to the 
```

Comment:
>> "Make any corrections/improvements that to the" -> "Make
corrections/improvements to the"
Done.

On Sat, Jun 13, 2015 at 12:29 PM, Ethan White <notifications@github.com>
wrote:

> In exercises/Tests-R.md
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/78#discussion_r32372204>
> :
>
> > @@ -0,0 +1,35 @@
> > +---
> > +layout: exercise
> > +title: Tests
> > +language: R
> > +---
> > +
> > +This is a follow up to the [Debugging problem]({{ site.baseurl }}/exercises/Debugging-R).
> > +
> > +Write tests for your 'Debugging' code for the following cases and save it in a
> > +file called `Tests-R.R`. Make any corrections/improvements that to the
>
> "Make any corrections/improvements that to the" -> "Make
> corrections/improvements to the"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/78/files#r32372204>
> .
>